tucson-teststand/GC_dev/docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: "3.6"
```

Comment:
Are double quotes really necessary here and for driver, max-file and max-size? I assumed yaml would handle those values correctly. I see you have them in the other yaml files in this PR as well.